website/cmdlets/index.html

```diff
@@ -10,7 +10,7 @@
     <ul>
       {% for product_hash in site.data.products %}
         {% assign product = product_hash.productName %}
-        <li><a href="#{{ product | slugify  }}">{{ product }}</a>
+        <li><a href="/google-cloud-powershell/cmdlets/{{ product | slugify  }}">{{ product }}</a>
```

Comment:
Why change the URL? Is the anchor (`#`) not correct? was this broken beforehand? etc.